### PR TITLE
refactor: move to RxJava3

### PR DESCRIPTION
**Pull request by jgiovaresco**
**Issue**

https://graviteedevops.atlassian.net/browse/COC-49

**Description**

BREAKING CHANGE: rxJava3 required

---
**Gravitee.io Automatic Deployment**

🚀 A prerelease version of this package has been published on Gravitee's private artifactory, you can:
 - use it directly by updating your project with version: `3.0.0-coc49-rxjava3-SNAPSHOT`
 - download it from Artifactory [here](https://odbxikk7vo-artifactory.services.clever-cloud.com/gravitee-snapshots/io/gravitee/cockpit/gravitee-cockpit-connectors/3.0.0-coc49-rxjava3-SNAPSHOT/gravitee-cockpit-connectors-3.0.0-coc49-rxjava3-SNAPSHOT.zip)

### CODE REVIEW

**Review by loriepisicchio — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -40,10 +40,10 @@
 
     <properties>
         <gravitee-bom.version>2.6</gravitee-bom.version>
-        <gravitee-node.version>1.25.0</gravitee-node.version>
+        <gravitee-node.version>2.0.0-bumpRxJava3-SNAPSHOT</gravitee-node.version>
```
Comment (marcambier) [inline pom.xml:43]: Could you please set it to the pre-released version ?
```suggestion
        <gravitee-node.version>2.0.0-alpha.3</gravitee-node.version>
```

File: pom.xml [hunk 2]
```diff
@@ -40,10 +40,10 @@
...
         <gravitee-plugin.version>1.24.1</gravitee-plugin.version>
         <gravitee-alert-api.version>1.9.1</gravitee-alert-api.version>
-        <gravitee-cockpit-api.version>1.13.3</gravitee-cockpit-api.version>
+        <gravitee-cockpit-api.version>2.0.0-bumpRxJava-SNAPSHOT</gravitee-cockpit-api.version>
```
Comment (marcambier) [inline pom.xml:46]: Could you please set it to the pre-released version ?
```suggestion
        <gravitee-cockpit-api.version>2.0.0-alpha.1</gravitee-cockpit-api.version>
```

**Review by marcambier — CHANGES_REQUESTED** [review]
Hello !
We made some pre-release versions of gravitee-node and cockpit-api.
So we would better use them than snapshots

**Review by marcambier — APPROVED** [review]

**Pull request merged**